### DISCUSSION TITLE
VG WORT

### DISCUSSION THREAD

**Discussion by LadySolveig** [StevenBlack/hosts#2284] [General]
I'm surprised to see that the VG WORT URLs are on the list. 
This is a well-known german institution that counts page views to authors and creators of texts and pays annual royalties for the available content from a minimum number of views.
I see no connection here with advertising, pornography or anything else worth protecting. The money that is distributed here annually comes from fixed levies by device manufacturers of storage media, copiers, etc. Libraries also pay levies here in Germany to support authors. 
Maybe someone here can give me a good reason to understand why blocking these urls is necessary. It reduces the well-earned income of the authors of texts in Germany by blocking.

![IMG_20230412_074225.jpg](https://user-images.githubusercontent.com/64533137/231361704-a209222c-e4c8-4bf0-8258-58ebef256729.jpg)

Further information about the [VG WORT](https://www.vgwort.de/die-vg-wort.html)

**Comment by hagezi** [comment]
It is a tracker https://whotracks.me/trackers/vg_wort.html
That's why you can find the domains in almost every blocklist.

```
Blocklists:
 - 1Hosts.Lite     BLOCKED
 - 1Hosts.Mini     BLOCKED
 - 1Hosts.Pro      BLOCKED
 - AdGuardDNS      BLOCKED
 - AhaDNS          BLOCKED
 - CONTROLD        BLOCKED
 - DNSforge.de     BLOCKED
 - EasyList        BLOCKED
 - HaGeZi.LIGHT    BLOCKED
 - HaGeZi.NORMAL   BLOCKED
 - HaGeZi.PERSONAL BLOCKED
 - HaGeZi.PRO      BLOCKED
 - HaGeZi.PRO.PLUS BLOCKED
 - HaGeZi.ULTIMATE BLOCKED
 - hBlock          BLOCKED
 - Lightswitch05   BLOCKED
 - NextDNS         BLOCKED
 - NoTracking      BLOCKED
 - OISD            BLOCKED
 - QuidsUp.NOTRACK BLOCKED
 - StevenBlack     BLOCKED
```

**Reply by LadySolveig** [reply]
Yes you're right, it recognizes and counts the visits and if the site ist visited from Germany so that the authors get money each year for publishing their content on the web if enough readers are interested in. 
I can understand you're point, but I think it must be very frustrating for publishers to see this on global blocklists. 
It is an anonymous tracking to be able to distribute the royalties. But of course I can also understand that this is difficult to understand outside of Germany and there is no reason to deal with the meaning and use. 
Many thanks for your response.
